bin/activate

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash
 ROOT_PREFIX=$(conda info --root)
```

Comment:
no -- it doesn't and and I borked the logic order (the code should have been in the `else` block).